src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3451,17 +3451,17 @@ namespace InternalHeatGains {
                 }
                 print(state.files.eio, "{},", GetScheduleName(state, state.dataHeatBal->People(Loop).WorkEffPtr));
 
-                if (state.dataHeatBal->People(Loop).ClothingType == 1) {
```

Comment:
Yep, I was moving a bit too quick to see that earlier.  I saw it wasn't just doing an assignment so I didn't do the array approach.  But obviously it's just looking up a string to perform that function call, so no problem.  If it's only used in this one place, do you prefer for me to declare the array local right here?  Or define it next to the enum so that it is updated if/when the enum changes?

Reply:
OK, this is the one remaining comment to address.  I'll fix this one up.

Reply:
Cleaned this one up, it's much better now!